### ISSUE TITLE
Improving label layout anchor for libreoffice?

### ISSUE THREAD

**Issue by bernhardreiter** [Intevation/intelmq-fody#43]
Libreoffice from Debian Version: 1:5.2.5-1~bpo8+1 does not honor the `text-anchor="middle"`
or `text-anchor="end"` SVG attributes resulting in bad axis labeling when exporting an svg from 921ef6a4054b7f65e002e8caee659088fccb8ce8

See both problems in the following screenshot:
![libreoffice5 2 5-d3js-svg-import](https://cloud.githubusercontent.com/assets/8190008/24393683/46ea5d36-1399-11e7-8f0a-44da2efb6d66.png)

### possible solutions
Check if never or other libreoffice versions have this issue fixed.

Try to either export the SVG without relying on text-anchor or change the SVG towards this.

**Comment by dmth** [comment]
In Writer they are shown centered. (lo  5.2.5.1)

**Comment by bernhardreiter** [comment]
True, when _inserting_ the diagram (as opposed to _opening_ it in libreoffice draw) the horizontal text-anchor position is fine, tested in the LO apps writer, calc, impress and draw.

The vertical positioning is off, though:
![libreoffice5 2 5-d3js-svg-insert](https://cloud.githubusercontent.com/assets/8190008/24394768/8d8e45b4-139d-11e7-80ee-c75ae5fdfe73.png)

**Comment by bernhardreiter** [comment]
Tested versions of inkscape and convert (from imagemagick) seems to work fine.
